app/controllers/pagecontroller.php

```diff
@@ -85,6 +86,24 @@ function getAllLinks()
         }
     }
 
+    /**
+     * Deletes a page by id
+     * @param int $id
+     * @return void
+     * @throws \Exception
+     * @author Luko Pecotic
+     */
+    public function deletePage($id) {
+        try {
+            $this->service->deletePage($id);
+    
+            $this->respond(null, 204); 
```

Comment:
200 is logischer 204 gebruik je voornamelijk als je iets zoekt maar er niks te vinden is

Reply:
Ik zal het aanpassen, maar op stackoverflow vond ik dit dus vandaar dat ik 204 gebruikte : https://stackoverflow.com/questions/29545861/what-is-the-http-status-return-code-for-a-successful-delete-statement-in-rest